CONTRIBUTING.md

```diff
@@ -98,3 +126,13 @@ version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
 [version]: http://contributor-covenant.org/version/1/4/
 [github]: https://github.com/dealerdirect/phpcodesniffer-composer-installer/issues
 [prs]: https://github.com/dealerdirect/phpcodesniffer-composer-installer/pulls
+
+## Footnotes
+
+1. All settings needed for the changelog-generator are set in `.github_changelog_generator`
+
+2. A convenience script is present at `bin/generate-changelog.sh` that will install the changelog-generator if it is not present and run the appropriate `github_changelog_generator` command.
```

Comment:
```suggestion
2. A convenience script is present at `bin/generate-changelog.sh` that will install the changelog-generator, if it is not present, and run the appropriate `github_changelog_generator` command.
```

Reply:
Also: should the script be run from the project root ? from the `bin` directory ?

Reply:
Good point,  I'll mention that.